.github/workflows/release.yml

```diff
@@ -180,23 +196,59 @@ jobs:
       - name: Upload artifacts for ${{ matrix.os_type }} platform
         uses: actions/upload-artifact@b4b15b8c7c6ac21ea08fcf65892d2ee8f75cf882 # v4.4.3
         with:
-          name: ${{ env.artifact_name }}
+          name: ${{ needs.generate-tag.outputs.artifact_name }}
           if-no-files-found: error
           path: |
-            ./tmp/${{ env.artifact_name }}.tgz
+            ./tmp/${{ needs.generate-tag.outputs.artifact_name }}.tgz
+
+  generate-release-notes:
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PRERELEASE: ${{ contains(github.ref, '-rc') }}
```

Comment:
I think pre-release is beta, not rc, @alexsporn please confirm

Reply:
pre-release is currently `-alpha` and `-beta`, we are treating `-rc` targetting the Testnet as normal releases right now.

Reply:
fixed it